src/application/collections/windows.yaml

```diff
@@ -581,8 +581,7 @@ actions:
                         recommend: standard
                         code: |-
                             del /f /q %localappdata%\Temp\*
-                            rd /s /q "%WINDIR%\Temp"
-                            rd /s /q "%TEMP%"
+                            del /s /q "%WINDIR%\Temp\*"
```

Comment:
Possibly add /f argument to remove read only files to be consistent with line 583

Reply:
Yes all good for line 584